src/audio/host-zephyr.c

```diff
@@ -484,22 +484,18 @@ static uint32_t host_get_copy_bytes_normal(struct comp_dev *dev)
 	buffer_c = buffer_acquire(buffer);
 
 	/* calculate minimum size to copy */
-	if (dev->direction == SOF_IPC_STREAM_PLAYBACK) {
-		/* limit bytes per copy to one period for the whole pipeline
-		 * in order to avoid high load spike
-		 */
+	if (dev->direction == SOF_IPC_STREAM_PLAYBACK)
 		free_bytes = audio_stream_get_free_bytes(&buffer_c->stream);
-		copy_bytes = MIN(hd->period_bytes, MIN(avail_bytes, free_bytes));
-		if (!copy_bytes)
-			comp_info(dev, "no bytes to copy, %d free in buffer, %d available in DMA",
-				  free_bytes, avail_bytes);
-	} else {
+	else
 		avail_bytes = audio_stream_get_avail_bytes(&buffer_c->stream);
-		copy_bytes = MIN(avail_bytes, free_bytes);
-		if (!copy_bytes)
-			comp_info(dev, "no bytes to copy, %d avail in buffer, %d free in DMA",
-				  avail_bytes, free_bytes);
-	}
+
+	/* limit bytes per copy to one period for the whole pipeline
+	 * in order to avoid high load spike
+	 */
+	copy_bytes = MIN(hd->period_bytes, MIN(avail_bytes, free_bytes));
+	if (!copy_bytes)
+		comp_info(dev, "no bytes to copy, %d in buffer, %d in DMA",
+			  avail_bytes, free_bytes);
```

Comment:
for playback, avail_bytes is the bytes available in dma, free_bytes is the free bytes in downstream buffer; but for capture, avail_bytes is the bytes available in buffer, and free bytes is the free bytes in downstream DMA. so the log info here probably is not correct.

Reply:
it is just print some information help developer debug when there is a bug exists,  to say wrong is not proper, afterall, deleoper always add/remove logs when debug, just print avail_bytes and free_bytes, developer have the sense to justify printed information.